Server/API/rooms/patchJoin.go

```diff
@@ -0,0 +1,23 @@
+package rooms
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	roomsScripts "ishtaloo.io/DB/Scripts/rooms"
+	cookieUtils "ishtaloo.io/Utils/cookie"
+)
+
+func patchJoin(c echo.Context) (err error) {
```

Comment:
Maybe call it something like POST /rooms/join? I don't really think patch works here logically ( even though you are technically updating the room)

Reply:
Because we are technically updating, I think it should be a patch. Why doesn't work logically for you?

Reply:
When I think of PATCH room I'm thinking of something like changing things about the room, no the users list (which is technically part of the room object) think about it as if the DB was written in SQL

Reply:
I think POST /room/join is way more obvious